phylogenetic/rules/construct_phylogeny.smk

```diff
@@ -62,5 +61,5 @@ rule refine:
             --date-inference {params.date_inference:q} \
             --clock-filter-iqd {params.clock_filter_iqd:q} \
             --stochastic-resolve \
-          2> {log:q}
+          2>&1 | tee {log:q}
```

Comment:
[general commentary on rule refine]

Did you explore temporal rooting? It works better for me for the genome build and I think is the better direction given the strong clock signal

Reply:
By "temporal rooting", do you mean `--root oldest`? Or something else?

Reply:
Shifted to `--root best` here.